.github/dependabot.yml

```diff
@@ -12,15 +12,6 @@ updates:
   - 2. to review
   - dependencies
   ignore:
-  - dependency-name: com.squareup.okhttp3:okhttp
-    versions:
-    - ">= 3.13.a"
-    - "< 3.14"
-  - dependency-name: com.squareup.okhttp3:okhttp
-    versions:
-    - ">= 3.14.a"
-    - "< 3.15"
   - dependency-name: com.squareup.okhttp3:okhttp
     versions:
     - ">= 4.a"
```

Comment:
I left okhttp 4+ on ignore because it requires kotlin.
Since SingleSignOn only uses java atm I don't think it should include a kotlin library.